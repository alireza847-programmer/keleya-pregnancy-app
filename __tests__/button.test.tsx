import React from 'react';
import renderer from 'react-test-renderer';
import {test, expect, jest} from '@jest/globals';
import Button from 'components/uiElements/button';

test('renders correctly', () => {
  const tree = renderer
    .create(<Button onPress={() => {}} text="Title" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
