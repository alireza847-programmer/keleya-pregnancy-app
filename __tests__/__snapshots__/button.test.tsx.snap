// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": false,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#69c0ba",
      "borderRadius": 11.25,
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 0,
      "opacity": 1,
      "paddingVertical": 23.345,
      "width": 600,
    }
  }
>
  <Text
    color="#ffffff"
    style={
      [
        {
          "color": "#ffffff",
          "fontFamily": "MuseoSansRounded-500",
          "fontSize": 38.244916147262124,
          "marginTop": 0,
          "textAlign": "auto",
          "width": null,
        },
      ]
    }
    text="Title"
    typography={
      {
        "fontFamily": "MuseoSansRounded-500",
        "fontSize": 38.244916147262124,
      }
    }
    width={null}
  >
    Title
  </Text>
</View>
`;
